locale/es_AR/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,5440 @@
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: messages\n"
```

Comment:
Is the rationale that it is easier for translators to modify the original spanish translation to variants than to start from scratch?

Also.. it's weird.. it's like did we support the different spanish variants already or is this introducing them. On the one hand, you're adding the locale files ... so they weren't there.. but onb the other, there was already reference to those locales ... can you explain that?

Reply:
There were no references to the variants before, I'm adding them. Translator teams have expressed different preferences regarding what to do with the files, but the conclusion we reached when talking to @mathjazz (Pontoon lead) was to "seed" the new locales by copying the old file over and they'd sort them out in pontoon later.

Reply:
Got it. Probably worth dumping that in the "context" on the PR for posterity.

FYI there was a [reference](https://github.com/mozilla/addons-server/blob/3f154e06a92e9a2c0d85516eea3ef26f2c2e684e/src/olympia/core/languages.py#L149) to the language but I guess not sufficient enough to trigger creating a translation file for that locale.

Reply:
Done